sunpy/map/maputils.py

```diff
@@ -452,6 +456,9 @@ def _bresenham(*, x1, y1, x2, y2):
     return np.array(res)
 
 
+@deprecated("5.0", message="The extract_along_coord function is deprecated and may be removed in "
+                           "version 5.1. Use pixelate_coord_path, and then pass its output to "
```

Comment:
I originally had exactly that, but then I took it out.  It seemed like it could give the impression that the default behavior of `pixelate_coord_path()` (i.e., `bresenham=False`) was not the recommended way to use that function (e.g., that it gives inferior results).  Reproducibility aside, I think that most users would prefer `bresenham=False` over `bresenham=True`, but a message like that may mean that they never even try the default behavior.

I think I'll add a full sentence

Reply:
> pixelate_coord_path uses a different line algorithm by default, but you can specify bresenham=True to use the same line algorithm as extract_along_coord.